### PR TITLE
Refactor in conda manager

### PR DESCRIPTION
**Pull request by dmaluenda**
Simplifying the conda usage according to https://github.com/I2PC/xmipp/pull/276.

Two main things here:

1. We launch conda jobs just by launching common jobs with a proper environ.
2. We check the deepLearningToolkit is a simpler way.

### CODE REVIEW

File: xmipp3/base.py [hunk 1]
```diff
@@ -148,18 +148,13 @@ def validateDLtoolkit(self, errors=None, **kwargs):
...
         # Trying to import keras to assert if DeepLearningToolkit works fine.
         kerasError = False
-        # condaEnvName preference: kwargs > protocol default > general default
-        condaEnvName = kwargs.get('_conda_env', getattr(self, "_conda_env",
-                                  CondaEnvManager.CONDA_DEFAULT_ENVIRON))
-        env = CondaEnvManager.modifyEnvToUseConda(xmipp3.Plugin.getEnviron(),
-                                                  condaEnvName)
         try:
             # subprocess.call('which python', shell=True, env=env)
-            subprocess.check_output('python -W ignore::FutureWarning '
```
Comment (rsanchezgarc) [inline xmipp3/base.py:158]: Why did you removed the -W ignore::FutureWarnings? If you remove it, they will appear messages such as feature X will be deprecated, that are not of interests for the user

Comment (dmaluenda) [reply xmipp3/base.py:158]: Now is always removed in the environ construction.

https://github.com/I2PC/xmipp/blob/8296fb03c09a61da5dea8d7d07fde027c715b974/src/xmipp/bindings/python/xmipp_base.py#L242

Comment (dmaluenda) [reply xmipp3/base.py:158]: Regarding which module we should import to check the correct installation, we can choose tensorflow or any other. It's just a quick check. Even though, we could check if the target (yml) is in the deepLearningToolkit directory, even if the environ exist. I mean, one of this checks is enough.

We shouldn't be crazy on this ;-p

Comment (rsanchezgarc) [reply xmipp3/base.py:158]: Ok. 
 I am fine with it. The problem is that Micrograph Cleaner does not use keras, only tensorflow and as we are checking for keras, the validation fails unless you install keras together with micrographCleaner.

**Comment by dmaluenda** [comment]
I will merge this

**Pull request merged**